### ISSUE TITLE
Create an app logo

### ISSUE THREAD

**Issue by trex** [trex/hawthorn-client#11]
A logo would be nice for the application header and favicon, and for beginning to develop a brand.

**Comment by moonquay** [comment]
I agree! We could even have a merch section that also sells stickers/shirts with consent- or other posi- topics :)
